tests/tests_files_elasticsearch_use_local_cert.yml

```diff
@@ -0,0 +1,86 @@
+- name: Ensure that the role runs with parameters from imfile to two elasticsearch outputs with use_local_omelasticsearch_cert
+  hosts: all
+  become: true
+  vars:
+    __test_inputfiles_dir: /var/log/inputdirectory
+    __test_inputfiles_conf: /etc/rsyslog.d/90-input-files-files_input.conf
+    __test_outputfiles_conf: /etc/rsyslog.d/31-output-elasticsearch-elasticsearch_output.conf
+    __test_ca_cert: /tmp/es-ca.crt
+    __test_cert: /tmp/es-cert.pem
+    __test_key: /tmp/es-key.pem
+
+  tasks:
+    - name: generate test files
+      copy:
+        dest: "{{ item }}"
+        content:
+          This is a fake {{ item }}.
+      delegate_to: localhost
+      loop:
+        - "{{ __test_ca_cert }}"
+        - "{{ __test_cert }}"
+        - "{{ __test_key }}"
+
+    - name: deploy config to send to elasticsearch
+      vars:
+        use_local_omelasticsearch_cert: true
+        logging_outputs:
+          - name: elasticsearch_output
+            type: elasticsearch
+            server_host: logging-es
+            server_port: 9200
+            index_prefix: project.
+            input_type: ovirt
+            retryfailures: off
+            ca_cert_src: "{{ __test_ca_cert }}"
+            cert_src: "{{ __test_cert }}"
+            key_src: "{{ __test_key }}"
+        logging_inputs:
+          - name: files_input
+            type: files
+            rsyslog_input_log_path: "{{ __test_inputfiles_dir }}/*.log"
+        logging_flows:
+          - name: flow_0
+            inputs: [files_input]
+            outputs: [elasticsearch_output, elasticsearch_output_ops]
+      include_role:
+        name: linux-system-roles.logging
+
+    - include: set_rsyslog_variables.yml
```

Comment:
See the notes in https://docs.ansible.com/ansible/latest/modules/include_module.html#notes regarding nonintuitive behavior and possible future deprecation.